app/Jobs/SetupAccount.php

```diff
@@ -1607,4 +1608,55 @@ private function addPostTemplates(): void
             'can_be_deleted' => true,
         ]);
     }
+
+    private function addReligions(): void
+    {
+        DB::table('religions')->insert([
+            [
+                'account_id' => $this->user->account_id,
+                'name' => trans('account.religion_christian'),
```

Comment:
We absolute must *not* put translated values in the database.
You can add a key, and use the key to get the translated value at runtime, but adding the translated value is a no go.

Reply:
I KNOW, in principle.
However, considering that each value can be either renamed, or deleted, how do you handle that? It can't be translated at runtime.
The only solution is to have an additional column with the default translation key, and have a field that can store the  overwritten value it if we rename the field.

Reply:
There are other solutions, yes.
I should be able to create an account in any language, then switch to another language, and everything should be translated.

Reply:
This is the ideal solution, but yes, it's more complex. This is why I've taken the easier route and did this.
It's more work to support what you want, and yes, what you want is the ideal solution.

We have the same problem with every single option under the Settings tab, by the way. At least 15 other pages of settings.

Reply:
Fixed and improved.

Reply:
Fixed and improved.